### ISSUE TITLE
Image download wont work

### ISSUE THREAD

**Issue by XDred** [DominikPieper/obsidian-ReadItLater#62]
Try to save page https://habr.com/ru/post/508672/
Console error: 
```
Access to fetch at 'https://habrastorage.org/r/w1560/webt/ps/6w/9v/ps6w9vwyfmogo3ihn8o6kbmimv0.png' from origin 'app://obsidian.md' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

Also, if `Download images to article folder` is enabled, page won`t save at all. 
Error: 
```
app.js:1 Uncaught (in promise) Error: File name cannot contain any of the following characters: * " \ / < > : | ?
    at Be (app.js:1:250004)
    at t.checkPath (app.js:1:1103614)
    at t.<anonymous> (app.js:1:1106275)
    at app.js:1:235505
    at Object.next (app.js:1:235610)
    at app.js:1:234547
    at new Promise (<anonymous>)
    at g (app.js:1:234292)
    at t.createFolder (app.js:1:1106165)
    at eval (plugin:obsidian-read-it-later:57:21)
```

Plugin version: 0.0.17

**Comment by lukacovicadam** [comment]
I can confirm the bug related to CORS.

However second issue with folder isn't caused by plugin. Even when I tried to create folder manually I got same error. I suggest raising issue in [Obsidian Forum](https://forum.obsidian.md/)

**Comment by XDred** [comment]
The second issue is related to https://github.com/DominikPieper/obsidian-ReadItLater/issues/46
Characters are not escaped when creating a folder.

**Comment by lukacovicadam** [comment]
Did you try to create new folder manually with same name as page title? Did it work?

**Comment by XDred** [comment]
This won't work as it is due to file system limitations.
Character escaping is present when creating files with the plugin, but not when creating folders using the "Download images to article folder" setting.

**Comment by lukacovicadam** [comment]
You're right, I've overlooked it. Fix will be in next release.

**Comment by lukacovicadam** [comment]
Fixed in [0.0.18](https://github.com/DominikPieper/obsidian-ReadItLater/releases/tag/0.0.18)
